locales/templates/app.pot

```diff
@@ -0,0 +1,622 @@
+#. Zerda Need l10n
+msgctxt "label_menu_download"
+msgid "Downloads"
+msgstr ""
+
+msgctxt "label_menu_clear_cache"
+msgid "Clear cache"
+msgstr ""
+
+msgctxt "label_menu_history"
+msgid "History"
+msgstr ""
+
+msgctxt "label_menu_share"
+msgid "Share"
+msgstr ""
+
+msgctxt "label_menu_my_shots"
+msgid "My shots"
+msgstr ""
+
+msgctxt "label_menu_capture_page"
+msgid "Capture page"
+msgstr ""
+
+msgctxt "label_menu_turbo_mode"
+msgid "Turbo mode"
+msgstr ""
+
+msgctxt "label_menu_block_ads"
+msgid "Block Ads"
+msgstr ""
+
+msgctxt "label_menu_block_image"
+msgid "Block image"
+msgstr ""
+
+msgctxt "label_menu_settings"
+msgid "Settings"
+msgstr ""
+
+#. GeolocationPermissionPrompt
+#, c-format
+msgctxt "geolocation_dialog_message"
+msgid ""
+"%1$s \n"
+"wants to use your device's location."
+msgstr ""
+
+msgctxt "geolocation_dialog_allow"
+msgid "ALLOW"
+msgstr ""
+
+msgctxt "geolocation_dialog_block"
+msgid "BLOCK"
+msgstr ""
+
+#. Setting Clear browsing data dialog
+msgctxt "setting_dialog_clear_data"
+msgid "Clear Data"
+msgstr ""
+
+msgctxt "setting_dialog_cancel"
+msgid "Cancel"
+msgstr ""
+
+msgctxt "setting_dialog_browsing_history"
+msgid "Browsing history"
+msgstr ""
+
+msgctxt "setting_dialog_search_history"
+msgid "Search history"
+msgstr ""
+
+msgctxt "setting_dialog_form_history"
+msgid "Form history"
+msgstr ""
+
+msgctxt "setting_dialog_cookies"
+msgid "Cookies"
+msgstr ""
+
+msgctxt "setting_dialog_cache"
+msgid "Cache"
+msgstr ""
+
+#. Download list
+msgctxt "no_downloads"
+msgid "No downloads"
+msgstr ""
+
+msgctxt "download_started"
+msgid "Downloading…"
+msgstr ""
+
+msgctxt "download_file_not_supported"
+msgid "File type not supported"
+msgstr ""
+
+msgctxt "cannot_find_the_file"
+msgid "Cannot find the file."
+msgstr ""
+
+msgctxt "cannot_delete_the_file"
+msgid "Cannot delete the file."
+msgstr ""
+
+#. Download status
+msgctxt "pause"
+msgid "Paused"
+msgstr ""
+
+msgctxt "pending"
+msgid "Pending"
+msgstr ""
+
+msgctxt "running"
+msgid "Downloading"
+msgstr ""
+
+msgctxt "successful"
+msgid "Successful"
+msgstr ""
+
+msgctxt "failed"
+msgid "Failed"
+msgstr ""
+
+msgctxt "unknown"
+msgid "Unknown"
+msgstr ""
+
+#. Setting data saving path dialog
+msgctxt "setting_dialog_sd_card"
+msgid "SD card"
+msgstr ""
+
+msgctxt "setting_dialog_internal_storage"
+msgid "Internal storage"
+msgstr ""
+
+#. BrowsingHistoryActivity
+msgctxt "browsing_history_clear_btn"
+msgid "CLEAR BROWSING HISTORY"
+msgstr ""
+
+msgctxt "browsing_history_empty_view_msg"
+msgid "No history"
+msgstr ""
+
+msgctxt "browsing_history_date_today"
+msgid "Today"
+msgstr ""
+
+msgctxt "browsing_history_date_yesterday"
+msgid "Yesterday"
+msgstr ""
+
+msgctxt "browsing_history_menu_delete"
+msgid "Delete"
+msgstr ""
+
+msgctxt "browsing_history_dialog_confirm_clear_message"
+msgid "Clear all browsing history?"
+msgstr ""
+
+msgctxt "browsing_history_dialog_btn_clear"
+msgid "Clear"
+msgstr ""
+
+msgctxt "remove_from_list"
+msgid "Remove from list"
```

Comment:
What's the object removed? Please explain in a comment

Reply:
Will do update the comment.
It's the item in download panel.